plugins/welcome.lua

```diff
@@ -0,0 +1,124 @@
+-- mod-version:3 --lite-xl 2.1
+
+local core = require "core"
+local style = require "core.style"
+local command = require "core.command"
+local keymap = require "core.keymap"
+local View = require "core.view"
+local common = require "core.common"
+local EmptyView = require "core.emptyview"
+local Node = require "core.node"
+
+local PluginManager = require "plugins.plugin_manager"
+local PluginView = require "plugins.plugin_manager.plugin_view"
+
+local welcomed = system.get_file_info(USERDIR .. PATHSEP .. "welcomed") ~= nil
```

Comment:
I wonder if you should just return here, if `welcomed` is true.
You could even do it as the first thing this plugin does, even before the `require`s.

Reply:
True! Will do that.